apps/condo/domains/contact/components/CreateContactForm.tsx

```diff
@@ -53,6 +55,7 @@ export const CreateContactForm: React.FC = () => {
     const UnitLabel = intl.formatMessage({ id: 'field.Unit' })
     const PropertyErrorMessage = intl.formatMessage({ id: 'field.Property.requiredError' })
     const UnitErrorMessage = intl.formatMessage({ id: 'field.Unit.requiredError' })
+    const roleLabel = intl.formatMessage({ id: 'ContactRole' })
```

Comment:
```suggestion
    const RoleLabel = intl.formatMessage({ id: 'ContactRole' })
```

Reply:
Why? This is just local variable.

Reply:
we use CamelCase for translation variables. Do you really want to change this style?

Reply:
ok, commits reverted. New translation variables renamed in ~~Pascal~~CamelCase.